test/test_pebble.py

```diff
@@ -479,6 +479,25 @@ def test_checks(self):
         with self.assertRaises(AttributeError):
             plan.checks = {}  # type: ignore
 
+    def test_log_targets(self):
+        plan = pebble.Plan('')
+        self.assertEqual(plan.log_targets, {})
+
+        location = "https://example.com:3100/loki/api/v1/push"
+        plan = pebble.Plan(
+            'log-targets:\n baz:\n  override: replace\n  type: loki\n  '
+            f'location: {location}')
```

Comment:
I realize you just copied my style here :-), but looking at this now I think it'd be significantly simpler to read the indentation on this if it were a multi-line string:

```python
    plan = pebble.Plan(f"""
log-targets:
  baz:
    override: replace
    type: loki
    location: {location}
""")

I think we should change this one and the other calls to `pebble.Plan(s)` in this file to keep things consistent.